sklearn/ensemble/bagging.py

```diff
@@ -65,7 +65,7 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
             print("building estimator %d of %d" % (i + 1, n_estimators))
 
         random_state = check_random_state(seeds[i])
-        seed = check_random_state(random_state.randint(MAX_INT))
+        seed = random_state.randint(MAX_INT)
```

Comment:
and also to ensure that trees built in a single fit or trees built with multiple small calls with warm_restart will be the same (i.e. generated from the same seed)